app/assets/stylesheets/pdf.css

```diff
@@ -1,51 +1,80 @@
-@import url("https://fonts.googleapis.com/css?family=Fira+Sans");
+.page-break {
+  page-break-after: always;
+  page-break-inside: avoid;
+  /*overflow: hidden;*/
 
-.pdf-page-break {
-  page-break-before: always;
+}
+
+@media print {
+  body {
+    -webkit-print-color-adjust: exact !important;
+    overflow: visible;
+  }
+  .batch {
+    /*border: 10px solid #b10d1e;*/
+  }
+
+  .batch-container {
+    /*background-color: #336699;*/
+    font-size: 16px;
+    width: 100%;
+  }
+
+  .batches-heading {
+
+  }
+
+  .heading h3 {
+  }
 }
 
 body {
   margin: 0;
 }
 
-.container {
-  margin: 66px 60px 0;
-  font-family: "Fira Sans", sans-serif;
-  font-size: 16px;
-  color: #000000;
+.batch {
+  display: flex;
+  flex-direction: column;
+  float: none;
+}
+
+.batch-container {
+  float: none;
+  margin: 20px;
+}
+
+.rebate_container {
+  float: none;
 }
 
 .border {
   border: 1px solid #b10d1e;
 }
 
-.container-secondary {
-  margin: 30px 60px 0;
+.rebate-container-secondary {
+  float: none;
+  margin: 20px;
   position: relative;
 }
 
-.main-header {
+.heading {
   background-color: #000000;
-  position: relative;
+  display: inline-block;
   height: 100px;
+  width: 100%;
 }
 
-.heading {
-  font-family: "Fira Bold", sans-serif;
+.heading h3 {
+  color: #ffffff;
```

Comment:
could this be var(--white)

Reply:
Variables didn't work when it was a PDF, and I forgot to put them in now that it's an html. Thanks for noticing!